### ISSUE TITLE
Other library implementation? 

### ISSUE THREAD

**Issue by shreyaskamathkm** [huangleiBuaa/CenteredWN#2]
Hi,

I was wondering if centered weight normalization technique will be available in other libraries such as pytorch or tensorflow?

Thank you!

**Comment by huangleiBuaa** [comment]
Dear shreyaskamathkm,
sorry for the late reply, since I did not manage my github for a long time.    Currently, I didn't  implement it on pytorch or tensorflow. However, I think it's easier to implement it on pytorch and tensorflow, compared to torch, since there is no need to write the back-propagtation, and one can follow the forward  to implement.
